src/api/Infrastructure/Shrooms.ModelMappings/Profiles/Other.cs

```diff
@@ -46,6 +48,13 @@ private void CreateKudosLogDtoMappings()
             CreateMap<KudosLog, UserKudosInformationDTO>();
         }
 
+        private void CreateKudosWelcomeMappings()
+        {
+            CreateMap<KudosWelcomeDTO, WelcomeKudosViewModel>();
```

Comment:
Can it be more consistent? I.e. WelcomeKudosDTO -> WelcomeKudosViewModel (or KudosWelcomeDTO -> KudosWelcomeViewModel although it sounds more strange).